examples/data/priorApproval.ts

```diff
@@ -772,6 +772,21 @@ export const validPriorApproval: Schema = {
     service: {
       flowId: 'c6628103-c648-4663-81e1-bfa0a1a18340',
       url: 'https://www.editor.planx.dev/southwark/apply-for-prior-approval/preview',
+      files: {
+        required: [
+          {
+            value: 'otherDrawing',
+            description: 'Other - drawing',
+          },
+        ],
+        recommended: [],
+        optional: [
+          {
+            value: 'sitePlan.proposed',
+            description: 'Site plan - proposed',
+          },
+        ],
+      },
```

Comment:
nit: swapping the `required` & `optional` would be more realistic here until we get around to re-generating example payloads in full!